src/zenml/new/pipelines/pipeline.py

```diff
@@ -900,9 +965,8 @@ def _register(
         registered_pipeline = client.zen_store.create_pipeline(
             pipeline=request
         )
-        logger.info(
-            "Registered pipeline `%s` (version %s).",
-            registered_pipeline.name,
+        logger.warning(
```

Comment:
I'm not really a fan of this: This is using a warning message in a completely misleading way just to highlight the colors and make it look pretty? This message will be printed for most pipeline runs and is definitely not a warning in any way. If someone were to set their log level to `warning`, they would now be shown this log and think it's something they need to worry about.

Reply:
The same thing also applies to other logs (at least in this file, e.g. the dashboard one).

Reply:
I have changed them back to info messages.